recipes/sharppy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "SHARPpy" %}
+{% set version = "v1.4.0b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sharppy/SHARPpy/archive/{{version}}.tar.gz
+  sha256: 3c780c441b3b090ebe50065595e4d94573c79d1bc77c7c71fd3e3869516708b5
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv --ignore-installed --no-cache-dir"
+  osx_is_app: True
+  entry_points:
+    - sharppy = runsharp.full_gui:main
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - pip
```

Comment:
I think you don't need setuptools, if you have pip

Reply:
Just removed it.  I should be pushing a commit with these changes in a second.